frontend/lint/tests/no-unscoped-text-selectors.unit.spec.js

```diff
@@ -1,6 +1,5 @@
-/* eslint-disable import/no-commonjs */
-const RuleTester = require("eslint").RuleTester;
-const rule = require("./no-unscoped-text-selectors");
+import { RuleTester } from "eslint";
```

Comment:
Convert to module import so it's more consistent with other tests. The reason it wasn't working in the first place is when we tried to change this file and husky would complain.

You could try to run `eslint --rulesdir frontend/lint/eslint-rules --max-warnings 0 --fix` and it seems like we shouldn't include the test files inside a directory passed to `rulesdir`. The [eslint repo](https://github.com/eslint/eslint) also have their tests in a different directory than the actual code, so I follow that convention.